docs/TRANSFORMERS.md

```diff
@@ -10,4 +10,38 @@ You can do it with `transformers` visitors.
 
 ## Usage
 
-TBD
\ No newline at end of file
+First of all you will need to create a file with the custom rules and transformers. For example,
+
+```js
+export const id = 'customTransformers';
```

Comment:
Does the file need to have an `id`? What is the meaning of it? How it is used elsewhere?

Reply:
Yes. It must to have `id`. plugin id, it is used to prefix all the rules of this plugin so they won't clash with built-in in rules